Bad <init> method call from inside of a branch

```
Information:Gradle tasks [:app:assembleDebug]
:app:preBuild UP-TO-DATE
:app:preDebugBuild UP-TO-DATE
:app:checkDebugManifest
:app:preReleaseBuild UP-TO-DATE
:app:prepareComAndroidSupportAppcompatV72301Library UP-TO-DATE
:app:prepareComAndroidSupportRecyclerviewV72301Library UP-TO-DATE
:app:prepareComAndroidSupportSupportV132301Library UP-TO-DATE
:app:prepareComAndroidSupportSupportV42301Library UP-TO-DATE
:app:prepareComCrashlyticsSdkAndroidAnswers132Library UP-TO-DATE
:app:prepareComCrashlyticsSdkAndroidBeta113Library UP-TO-DATE
:app:prepareComCrashlyticsSdkAndroidCrashlytics252Library UP-TO-DATE
:app:prepareComCrashlyticsSdkAndroidCrashlyticsCore235Library UP-TO-DATE
:app:prepareComGithubChrisbanesPhotoviewLibrary124Library UP-TO-DATE
:app:prepareComGoogleAndroidGmsPlayServicesBase780Library UP-TO-DATE
:app:prepareComGoogleAndroidGmsPlayServicesLocation780Library UP-TO-DATE
:app:prepareComGoogleAndroidGmsPlayServicesMaps780Library UP-TO-DATE
:app:prepareComMsSquareExpandableTextView014Library UP-TO-DATE
:app:prepareIoFabricSdkAndroidFabric136Library UP-TO-DATE
:app:prepareJpCoCyberagentAndroidGpuimageGpuimageLibrary123Library UP-TO-DATE
:app:prepareMeGrantlandAutofittextview021Library UP-TO-DATE
:app:prepareOrgApmemToolsLayouts19Library UP-TO-DATE
:app:prepareDebugDependencies
:app:compileDebugAidl UP-TO-DATE
:app:compileDebugRenderscript UP-TO-DATE
:app:generateDebugBuildConfig UP-TO-DATE
:app:generateDebugAssets UP-TO-DATE
:app:mergeDebugAssets UP-TO-DATE
:app:generateDebugResValues UP-TO-DATE
:app:generateDebugResources UP-TO-DATE
:app:mergeDebugResources UP-TO-DATE
:app:processDebugManifest UP-TO-DATE
:app:fabricGenerateResourcesDebug
:app:processDebugResources
:app:generateDebugSources
:app:processDebugJavaRes UP-TO-DATE
:app:compileDebugJavaWithJavac UP-TO-DATE
:app:compileDebugNdk UP-TO-DATE
:app:compileDebugSources UP-TO-DATE
:app:dexDebug UP-TO-DATE
:app:validateDebugSigning
:app:packageDebug
:app:zipalignDebug
:app:assembleDebug FAILED
Error:Execution failed for task ':app:assembleDebug'.
> Bad <init> method call from inside of a branch
Exception Details:
  Location:
    com/getkeepsafe/dexcount/PackageTree.<init>(Ljava/lang/String;)V @27: invokespecial
  Reason:
    Error exists in the bytecode
  Bytecode:
    0x0000000: b800 214d b800 359a 0006 a700 172a 2b2c
    0x0000010: 1236 3212 022b b900 3c03 00b7 002c a700
    0x0000020: 0f2a 2b2b b800 40b8 0029 b700 2cb1     
  Stackmap Table:
    append_frame(@13,Object[#67])
    same_frame(@33)
    full_frame(@45,{Object[#2],Object[#69],Object[#67]},{})
Information:BUILD FAILED
Information:Total time: 27.534 secs
Information:1 error
Information:0 warnings
Information:See complete output in console
```

------------------------------------------------------------
Gradle 2.4
------------------------------------------------------------

Build time:   2015-05-05 08:09:24 UTC
Build number: none
Revision:     5c9c3bc20ca1c281ac7972643f1e2d190f2c943c

Groovy:       2.3.10
Ant:          Apache Ant(TM) version 1.9.4 compiled on April 29 2014
JVM:          1.7.0_79 (Oracle Corporation 24.79-b02)
OS:           Windows 7 6.1 amd64

```
Executing tasks: [assembleDebug]

Configuration on demand is an incubating feature.
Incremental java compilation is an incubating feature.
:app:preBuild UP-TO-DATE
:app:preDebugBuild UP-TO-DATE
:app:checkDebugManifest
:app:preReleaseBuild UP-TO-DATE
:app:prepareComAndroidSupportAnimatedVectorDrawable2321Library UP-TO-DATE
:app:prepareComAndroidSupportAppcompatV72321Library UP-TO-DATE
:app:prepareComAndroidSupportCardviewV72321Library UP-TO-DATE
:app:prepareComAndroidSupportCustomtabs2321Library UP-TO-DATE
:app:prepareComAndroidSupportDesign2321Library UP-TO-DATE
:app:prepareComAndroidSupportRecyclerviewV72321Library UP-TO-DATE
:app:prepareComAndroidSupportSupportV42321Library UP-TO-DATE
:app:prepareComAndroidSupportSupportVectorDrawable2321Library UP-TO-DATE
:app:prepareComFacebookAndroidFacebookAndroidSdk4110Library UP-TO-DATE
:app:prepareComGoogleAndroidGmsPlayServicesBase840Library UP-TO-DATE
:app:prepareComGoogleAndroidGmsPlayServicesBasement840Library UP-TO-DATE
:app:prepareComGoogleAndroidGmsPlayServicesMaps840Library UP-TO-DATE
:app:prepareDebugDependencies
:app:compileDebugAidl UP-TO-DATE
:app:compileDebugRenderscript UP-TO-DATE
:app:generateDebugBuildConfig UP-TO-DATE
:app:generateDebugAssets UP-TO-DATE
:app:mergeDebugAssets UP-TO-DATE
:app:generateDebugResValues UP-TO-DATE
:app:generateDebugResources UP-TO-DATE
:app:mergeDebugResources UP-TO-DATE
:app:processDebugManifest UP-TO-DATE
:app:processDebugResources UP-TO-DATE
:app:generateDebugSources UP-TO-DATE
:app:compileDebugJavaWithJavac UP-TO-DATE
:app:compileDebugNdk UP-TO-DATE
:app:compileDebugSources UP-TO-DATE
:app:prePackageMarkerForDebug
:app:transformClassesWithDexForDebug UP-TO-DATE
:app:mergeDebugJniLibFolders UP-TO-DATE
:app:transformNative_libsWithMergeJniLibsForDebug UP-TO-DATE
:app:processDebugJavaRes UP-TO-DATE
:app:transformResourcesWithMergeJavaResForDebug UP-TO-DATE
:app:validateDebugSigning
:app:packageDebug UP-TO-DATE
:app:zipalignDebug UP-TO-DATE
:app:assembleDebug FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:assembleDebug'.
> Bad <init> method call from inside of a branch
Exception Details:
  Location:
    com/getkeepsafe/dexcount/PackageTree.<init>()V @85: invokespecial
  Reason:
    Error exists in the bytecode
  Bytecode:
    0x0000000: 04bd 0004 5903 0112 6cb8 007b 5359 10ff
    0x0000010: 1202 b800 7f2a 5fab 0000 009e 0000 0005
    0x0000020: aad3 b1ff 0000 0031 c861 6941 0000 0044
    0x0000030: ef05 6b77 0000 0059 0000 9b75 0000 0078
    0x0000040: 0756 9009 0000 0081 5f5a 5903 32b8 005e
    0x0000050: c000 605f 57b7 0082 a700 675f 5a59 0332
    0x0000060: 126c b800 86c0 006c 5f57 b700 88a7 0052
    0x0000070: 5f5a 5903 32b8 005e c000 605f 5904 3212
    0x0000080: 6cb8 0086 c000 6c5f 57b7 008a a700 335f
    0x0000090: 5a57 b700 8ba7 002a 5f5a 5903 325f 5904
    0x00000a0: 325f 5905 3212 6cb8 0086 c000 6c5f 57b7
    0x00000b0: 002d a700 0dbb 008d 5912 8fb7 0090 bf57
    0x00000c0: b1                                     
  Stackmap Table:
    full_frame(@72,{UninitializedThis},{Object[#146],UninitializedThis})
    full_frame(@91,{UninitializedThis},{Object[#146],UninitializedThis})
    full_frame(@112,{UninitializedThis},{Object[#146],UninitializedThis})
    full_frame(@143,{UninitializedThis},{Object[#146],UninitializedThis})
    full_frame(@152,{UninitializedThis},{Object[#146],UninitializedThis})
    full_frame(@181,{UninitializedThis},{Object[#146],UninitializedThis})
    full_frame(@191,{Object[#2]},{Object[#146]})


* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 3.279 secs

```
Happened again to me OTL

--- a/src/main/groovy/com/getkeepsafe/dexcount/PackageTree.groovy
+++ b/src/main/groovy/com/getkeepsafe/dexcount/PackageTree.groovy
@@ -49,11 +49,15 @@ class PackageTree {
     // nodes and possibly non-empty for class nodes.
     private final Set<HasDeclaringClass> fields_ = new HashSet<>()
 
-    PackageTree(Deobfuscator deobfuscator = null) {
+    PackageTree() {
+        this("", false, null)
+    }
+
+    PackageTree(Deobfuscator deobfuscator) {
         this("", false, deobfuscator)
     }
 
-    PackageTree(String name, Deobfuscator deobfuscator = null) {
+    PackageTree(String name, Deobfuscator deobfuscator) {
         this(name, isClassName(name), deobfuscator)
     }
 

